src/main/java/com/twilio/taskrouter/application/servlet/EventsServlet.java

```diff
@@ -25,48 +31,85 @@
 @Singleton
 public class EventsServlet extends HttpServlet {
 
-  public static final String LEAVE_MSG = "Sorry, All agents are busy. Please leave a message. "
+  protected static final String LEAVE_MSG = "Sorry, All agents are busy. Please leave a message. "
```

Comment:
Do we intend to use `LEAVE_MSG`, `OFFLINE_MSG` outside the scope of this class through inheritance? If the answer is no, would you mind make them private please?